content/en/tracing/setup/dotnet.md

```diff
@@ -286,21 +287,20 @@ ENV DD_INTEGRATIONS=/opt/datadog/integrations.json
+-----------------------------| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+`DD_DISABLED_INTEGRATIONS`   | `DisabledIntegrationNames` | Sets a list of integrations to disable. All other integrations remain enabled. If not set, all integrations are enabled. Supports multiple values separated with semicolons. Valid values are the integration names listed in the [Integrations][3] section below. |
+`DD_TRACE_ANALYTICS_ENABLED` | `AnalyticsEnabled`         | Shorthand that enables default Trace Search and Analytics settings for web framework integrations. Valid values are: `true` or `false` (default).                                                                                                                  |
 
-Environment Variable or AppSetting       | Property Name              | Description                                                                                                                                                                                                                                                        |
----------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-`DD_DISABLED_INTEGRATIONS`               | `DisabledIntegrationNames` | Sets a list of integrations to disable. All other integrations remain enabled. If not set, all integrations are enabled. Supports multiple values separated with semicolons. Valid values are the integration names listed in the [Integrations][3] section below. |
-`DD_TRACE_ANALYTICS_ENABLED`             | `AnalyticsEnabled`         | Shorthand that enables default Trace Search and Analytics settings for web framework integrations. Valid values are: `true` or `false` (default).                                                                                                                  |
+The following column lists integration-specific settings. The first column indicates the name used in environment variables or configuration files. The second column indicates the name of the propery on the `IntegrationSettings` class. You can access these properties in code through `Tracer.Instance.Settings.Integrations["<INTEGRATION>"]`. Integration names are listed in the [Integrations][3] section below.
```

Comment:
"The following table"

Reply:
Oops. 85a58fd59d34a22d28485de09d01d06b2dd4b383